core.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-08/schema#",
+    "$id": "https://json-schema.org/draft-08/core",
+    "$recursiveAnchor": true,
+    "$vocabulary": {
+        "https://json-schema.org/draft-08/vocabularies/core": true
+    },
+
+    "title": "Core vocabulary meta-schema",
+    "type": ["object", "boolean"],
+    "properties": {
+        "$id": {
+            "type": "string",
+            "format": "uri-reference"
+        },
+        "$schema": {
+            "type": "string",
+            "format": "uri"
+        },
+        "$ref": {
+            "type": "string",
+            "format": "uri-reference"
+        },
+        "$recursiveRef": {
+            "type": "string",
+            "format": "uri-reference"
+        },
+        "$recursiveAnchor": {
+            "type": "boolean"
```

Comment:
This should be

```json
{
  "const": true,
  "default": false
}
```

Reply:
This is probably where you remember adding it 😄

Reply:
@gregsdennis I think you fixed this elsewhere?  Regardless, I'll do a separate PR for meta-schema cleanup.  I always have to audit that and the change log at the end of the draft process.

Reply:
I fixed the meta-schema not broken out in PR https://github.com/json-schema-org/json-schema-spec/pull/690

Reply:
@gregsdennis OK, let me re-base and that will be a merge conflict and I'll sort it out.

Reply:
@gregsdennis brain not working well today apparently.  I eventually realized this was a duplication, merged the two descriptions (it uses `default` and `type` now) and made the "regular" meta-schema recursively extend core, which is kind of the point of having it.  That should solve it.  I hope.

Reply:
@gregsdennis it's just nicer.  Some systems for things other than validation work much better with `type` and may or may not be able to infer the type from `const`.  Yes, I know it's trivial in this case, it's just nicer to be consistent in such a visible schema.